packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -148,88 +152,81 @@ export function useTreeItem_unstable<Value = string>(
     }
   });
```

Comment:
Would be nice to change this:

```ts
      case End:
        return requestNavigation({ event, value, type: treeDataTypes.end, target: event.currentTarget });
      case Home:
        return requestNavigation({ event, value, type: treeDataTypes.home, target: event.currentTarget });
      case ArrowUp:
        return requestNavigation({ event, value, type: treeDataTypes.arrowUp, target: event.currentTarget });
      case ArrowDown:
        return requestNavigation({ event, value, type: treeDataTypes.arrowDown, target: event.currentTarget });
```

To this:

```ts
      case End:
      case Home:
      case ArrowUp:
      case ArrowDown:
        return requestNavigation({ event, value, type: treeDataTypes[event.key], target: event.currentTarget });
```

Reply:
Not equivalent though, I'll need to modify `arrowDown` to `ArrowDown`, let's keep this for another PR